http/src/main/java/io/micronaut/http/form/FormConfigurationProperties.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.form;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+
+@ConfigurationProperties("micronaut.form")
```

Comment:
rename prefix to `micronaut.http.forms`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/11272/commits/7be3b6be79c7d2ddd865db81c15686362bd71a63